src/adr/adr-cli.csproj

```diff
@@ -4,6 +4,12 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <RuntimeIdentifiers>win-x64;win-x86;linux-x64;osx-x64</RuntimeIdentifiers>    
     <StartupObject>adr.Program</StartupObject>
+    <Description>A Windows equivalent of adr-tools (https://github.com/npryce/adr-tools)</Description>
+    <PackageLicenseExpression>MIT License</PackageLicenseExpression>
+    <AssemblyVersion>1.1.0.0</AssemblyVersion>
+    <FileVersion>1.1.0.0</FileVersion>
```

Comment:
Just the `<Version>` tag is sufficient

Reply:
If in future you'll want to distribute it with an MSI package (specially in old Windows versions) you'll need it when distributing an update

Reply:
I just take my queue from here: [the first note](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#properties-per-attribute) and it says that `Version` is the one to give all other "Version" attributes their sane default values in sdk style projects

Reply:
ok, probably this aspect is changed in .NET Core from .NET Framework